packages/widgets/user-management-widget/src/lib/widget/state/selectors.ts

```diff
@@ -57,6 +57,12 @@ export const getSelectedUsersUserIds = createSelector(
   getSelectedUsers,
   (users) => users.map((user) => user.userId),
 );
+export const getSelectedUsersMap = createSelector(getSelectedUsers, (users) =>
```

Comment:
I think we should give it a more meaningful name, maybe something like `getSelectedUsersIds`